src/Castle.Core.Tests/DynamicProxy.Tests/CustomModifiersTestCase.cs

```diff
@@ -150,7 +150,7 @@ public void ReflectionReturnsCorrectModoptOnParamTypeForGeneratedType(string typ
 
 			var modopts = this.generatedTypes[typeName].GetMethod("Foo").GetParameters()[0].GetOptionalCustomModifiers();
 
-			CollectionAssert.AreEqual(expected: CustomModifiersTestCase.customModifiers[typeNameWithoutSuffix].Reverse(), actual: modopts);
+			CollectionAssert.AreEqual(expected: Enumerable.Reverse(CustomModifiersTestCase.customModifiers[typeNameWithoutSuffix]), actual: modopts);
```

Comment:
This is necessary because apparently, C# 14 changed the method group overload resolution such that `array.Reverse()` now resolves to `Span<>.Reverse` (which reverses in-place and returns `void`) instead of `Enumerable.Reverse` (which returns a new, reversed sequence).

See e.g. https://medium.com/c-sharp-programming/csharp-14-reverse-array-bug-cfe0742870e3.